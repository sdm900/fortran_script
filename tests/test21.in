x=1
if (x==2) print "opps"
if (x==1) print 'x= ',x

