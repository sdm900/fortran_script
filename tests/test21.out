x=      1.00000 
